notes/src/App.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import Note from './Components/Note';
+import './App.css';
+import noteData from './data';
+import Notes from './Components/Notes';
+class App extends Component {
+constructor(){
+  super();
+  this.state = {
+    note: []
+  };
+}
+
+componentDidMount(){
+  this.setState({ note: noteData});
```

Comment:
I like that you set this up pulling in notes from another file.  this will make a transition to getting info from a server that much easier.